### PR TITLE
Add omero-py python package

### PR DESCRIPTION
**Pull request by manics**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

## Background
This package was originally in bioconda, but we're moving it to conda-forge to support other packages, and also for Windows support: https://github.com/bioconda/bioconda-recipes/pull/15831#issuecomment-501633453

The ome has been maintaining it's own repository for this package due to problems with the bioconda package:
- https://github.com/ome/conda-omero-py
- https://anaconda.org/ome/omero-py

But now that the dependency zeroc-ice 3.6 is available in conda-forge we'd like to add omero-py here too.

### CODE REVIEW

**Comment by manics** [comment]
@jburel@ joshmoore @sbesson please comment to confirm you're happy to be maintainers (last item in the checklist)

**Comment by sbesson** [comment]
I am happy to be added as a maintainer of this recipe

**Comment by joshmoore** [comment]
I am happy to be added as a maintainer of this recipe

File: recipes/omero-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{% set name = "omero-py" %}
+{% set version = "5.8.3" %}
```
Comment (joshmoore) [inline recipes/omero-py/meta.yaml:2]: Could be bumped to 5.9.0 if you'd like.

**Comment by joshmoore** [comment]
@manics: can you close this so I can re-create?

**Comment by manics** [comment]
Done! Though you should have access to this branch `microscopepony:omero-py` anyway.

**Pull request closed without merging**